modules/cnf-about-numa-aware-scheduling.adoc

```diff
@@ -21,3 +21,8 @@ NodeResourceTopology API:: The `NodeResourceTopology` API describes the availabl
 NUMA-aware scheduler:: The NUMA-aware secondary scheduler receives information about the available NUMA zones from the `NodeResourceTopology` API and schedules high-performance workloads on a node where it can be optimally processed.
 Node topology exporter:: The node topology exporter exposes the available NUMA zone resources for each compute node to the `NodeResourceTopology` API. The node topology exporter daemon tracks the resource allocation from the kubelet by using the `PodResources` API.
 PodResources API:: The `PodResources` API is local to each node and exposes the resource topology and available resources to the kubelet.
++
+[NOTE]
+====
+The `List` endpoint of the `PodResources` API exposes exclusive CPUs allocated to a particular container. The API does not expose CPUs that belong to a shared pool.
```

Comment:
Since we are highlighting the `List` endpoint, perhaps we should mention the `GetAllocatableResources` endpoint as well given that we use that to learn about allocatable resource. You can refer to the related [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2043-pod-resource-concrete-assigments) here for details.